plan/new_physical_plan_builder.go

```diff
@@ -1090,26 +1090,80 @@ func (p *TopN) getChildrenPossibleProps(prop *requiredProp) [][]*requiredProp {
 	return props
 }
 
+func (p *LogicalAggregation) getStreamAggs() []PhysicalPlan {
+	for _, aggFunc := range p.AggFuncs {
+		if aggFunc.GetMode() == expression.FinalMode {
```

Comment:
This code seems never be touched ?

Reply:
We do `decompose` will have `FinalMode` aggregation.